src/v/wasm/wasmtime.cc

```diff
@@ -424,7 +424,11 @@ absl::flat_hash_map<ss::sstring, ss::sstring>
 make_environment_vars(const model::transform_metadata& meta) {
     absl::flat_hash_map<ss::sstring, ss::sstring> env = meta.environment;
     env.emplace("REDPANDA_INPUT_TOPIC", meta.input_topic.tp());
-    env.emplace("REDPANDA_OUTPUT_TOPIC", meta.output_topics.begin()->tp());
```

Comment:
Might want to throw that in the release notes for posterity

Reply:
Yeah, done. I honestly think it's probably confusing to anyone who reads that, but if they know about this random env var it will be clear. IYKYK I guess.